tensorboard/plugins/projector/vz_projector/scatterPlotVisualizerSprites.ts

```diff
@@ -61,6 +61,15 @@ const VERTEX_SHADER = `
     if (sizeAttenuation) {
       outputPointSize = -pointSize / cameraSpacePos.z;
     }
+    else {  // Create size attenuation (if we're in 2D mode)
+      const float shrinkFactor = 0.5;
```

Comment:
This is great. Since UI code is hard to test, can you attach screenshots of before and after size attenuation when zoomed in. Thank you!

Reply:
Thanks, I'll add some visuals in updated pull request. I'm reconsidering and simplifying the zoom function to reduce number of math calls. I'll split the pause-resume and sprite zoom functionalities and supersede with new pull requests.